### PR TITLE
Update bootstrap computed props

### PR DESCRIPTION
**Pull request by ashwiniHerle**
This PR refactors the following components to latest react-bootstrap:

- [x] components/computedProps/*
- [x] apps/mydb/elements/details/computeTasks/*
- [x] apps/mydb/elements/tabLayout/*
- [x] components/common/ClipboardCopyText.js
- [x] components/common/ConfirmModal.js

Also contains bugfix for the error when 'copy to metadata' button is clicked in research plans

### CODE REVIEW

**Review by JoachimTillessen — COMMENTED** [review]
Looks great!

File: app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTaskContainer.js [hunk 1]
```diff
@@ -67,30 +66,31 @@ export default class ComputeTaskContainer extends React.Component {
     const { tasks } = this.state;
 
     return (
-      <Panel className="eln-panel-detail" variant="primary">
-        <Panel.Heading>
-          {'Task'}
+      <Card className="eln-panel-detail">
```
Comment (JoachimTillessen) [inline app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTaskContainer.js:69]: I think the CSS related to that class might be broken now. See cellLines.scss

File: app/packs/src/apps/mydb/elements/tabLayout/TabLayoutCell.js [hunk 2]
```diff
@@ -44,20 +44,27 @@ class TabLayoutCell extends Component {
...
+        </p>
       </div>
     ) : (
       <div>
-        <i className={cellIcon} title={[cellTitle, cellDescription].join(': ')}>
+        <i
+          className={`${cellIcon} text-center ${isHidden ? 'text-gray-600' : 'text-primary'}`}
+          title={[cellTitle, cellDescription].join(': ')}
+        >
           {isHidden ? '\u00A0' : ''}
```
[the remark this answers is not in the document]
Comment (ashwiniHerle) [reply app/packs/src/apps/mydb/elements/tabLayout/TabLayoutCell.js:58]: It's just a non-breaking space. I had mistaken it as something else, removing it from the code.

File: app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTask.js
[the remark this answers is not in the document]
Comment (ashwiniHerle) [reply app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTask.js:1]: Yes, I think coming back later when every code is refactored makes more sense based on our timeline. I'll make a note of it in the spreadsheet.

File: app/packs/src/components/common/ClipboardCopyText.js [hunk 3]
```diff
@@ -19,10 +19,9 @@ export default class ClipboardCopyText extends Component {
         overlay={<Tooltip id="copy_clipboard">copy to clipboard</Tooltip>}
       >
         <span
-          style={{ display: 'inline' }}
           role="button"
           data-clipboard-text={clipText}
-          className="clipboardBtn"
+          className="clipboardBtn d-inline"
```
[the remark this answers is not in the document]
Comment (ashwiniHerle) [reply app/packs/src/components/common/ClipboardCopyText.js:24]: I think all of these codes use Clipboard library and this classname is used by that.

File: app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTaskContainer.js (hunk 1 above)
Comment (ashwiniHerle) [reply app/packs/src/apps/mydb/elements/details/computeTasks/ComputeTaskContainer.js:69]: Adapted the class based on header and have kept the old classes intact until upgrade is finished. Will make a note in spreadsheet to clean up once all codes are refactored

File: app/packs/src/components/common/ClipboardCopyText.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (JoachimTillessen) [reply app/packs/src/components/common/ClipboardCopyText.js:24]: Another dark secret uncovered – to me using this library in chemotion is wrong on many levels. The library was introduced to replace functionality from the flash plug-in (!!!). I don't see why anybody would choose to have an external lib extracting data from dom-elements in a react app, when you can copy internal component state directly to the clipboard, with a single line:
`navigator.clipboard.writeText(something)`
This is obviously out of scope for now, but I'll add it to the refactoring list.

[the remark this answers is not in the document]
Comment (JoachimTillessen) [reply app/packs/src/components/common/ClipboardCopyText.js:24]: It's already on there. :D

**Pull request merged**